src/components/barChart/barChart.html

```diff
@@ -0,0 +1,4 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.min.js"></script>
```

Comment:
as I remember, you should have installed it via npm. I can be mistaken

Reply:
++